game.libretro.mrboom/game.libretro.mrboom.txt

```diff
@@ -0,0 +1 @@
+game.libretro.mrboom https://github.com/kodi-game/game.libretro.mrboom.git master
```

Comment:
`https://github.com/kodi-game/game.libretro.mrboom`

Reply:
oops. I copied the URL from github and it thought the .git was a good idea. Should I fix?

Reply:
yes, it doesn't build otherwise

Reply:
already fixed